pkg/manager/member/pd_member_manager.go

```diff
@@ -292,6 +265,14 @@ func (pmm *pdMemberManager) syncTidbClusterStatus(tc *v1alpha1.TidbCluster, set
 	pdClient := pmm.pdControl.GetPDClient(tc)
 	healthInfo, err := pdClient.GetHealth()
 	if err != nil {
+		tc.Status.PD.Health = false
+		tc.Status.PD.Members = nil
```

Comment:
Can't set members to nil.
If the getHealth failed, it just indicates this request failed, not the whole PD cluster unhealthy.